app/views/users/show.html.erb

```diff
@@ -34,7 +34,7 @@
           <%end%>
           <%="#{@user.name} has no any post" unless !@user.posts.empty?%>
         </ul>
-        <%=button_to "See all posts", "/users/#{@user.id}/posts", class: 'all-posts', method: 'get'%>
+        <%=button_to "See all posts", "/users/#{@user.id}/posts", class: 'all-posts', method: :get%>
```

Comment:
- [ ] Currently, when the `See all posts ` button is clicked, it leads to an error page so kindly fix that as well :wink: 
- [ ] You can consider handling the case where there are no posts, that way a user will be navigated to the form to add a new post, and if there are posts they are Navigated to the `posts index` page :100: 
- Error Message

[Screencast from 08-25-2023 12:57:18 PM.webm](https://github.com/Mahamadou-Nouridine/blog-app/assets/102699958/fb08840f-df36-48d9-b073-89d5cf3c4b48)

- [ ] Once this is fixed, kindly ensure that your comment and like features work as expected so the next reviewer can check if they all work as expected :wink: